monitoring/configure-monitoring-wds.sh

```diff
@@ -19,10 +19,11 @@ set -e # exit on error
 
 ctx="kind-kubeflex"
 wds="wds1"
+monitoring_ns="ks-monitoring"
 
 while [ $# != 0 ]; do
     case "$1" in
-        (-h|--help) echo "$0 usage: ( --host-cluster-context (e.g., --host-cluster-context core-cluster (default value: kind-kubeflex)) | --space-name (e.g., --space-name wds1 (default value: wds1))*"
+        (-h|--help) echo "$0 usage: ( --host-cluster-context (e.g., --host-cluster-context core-cluster (default value: kind-kubeflex)) | --space-name (e.g., --space-name wds1 (default value: wds1)) | --monitoring-ns (e.g., --monitoring-ns ks-monitoring (default value: ks-monitoring))*"
```

Comment:
(A) I would document `-h` and `--help`.
(B) Missing close paren.

Reply:
Fixed!